src/components/atoms/AddCompareIcon/AddCompareIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import { Center, Icon } from '@chakra-ui/react'
+import { FC } from 'react'
+
+interface AddCompareIconProps {
+  onClick: () => void
+}
+
+export const AddCompareIcon: FC<AddCompareIconProps> = ({ onClick }) => {
+  return (
+    <Center border="1px solid #00242C" borderRadius="50%" width="32px" height="32px" onClick={onClick}>
+      <Icon viewBox="0 0 16 16" alignSelf={`center`}>
+        <path
+          d="M7.99977 12.5C7.85803 12.5 7.73931 12.4521 7.64358 12.3562C7.54786 12.2604 7.5 12.1417 7.5 12V8.49997H3.99998C3.85832 8.49997 3.73957 8.45203 3.64375 8.35617C3.54792 8.26029 3.5 8.14149 3.5 7.99977C3.5 7.85803 3.54792 7.73931 3.64375 7.64358C3.73957 7.54786 3.85832 7.5 3.99998 7.5H7.5V3.99998C7.5 3.85832 7.54793 3.73957 7.6438 3.64373C7.73968 3.54791 7.85848 3.5 8.0002 3.5C8.14193 3.5 8.26066 3.54791 8.35638 3.64373C8.45211 3.73957 8.49997 3.85832 8.49997 3.99998V7.5H12C12.1417 7.5 12.2604 7.54793 12.3562 7.6438C12.4521 7.73968 12.5 7.85848 12.5 8.0002C12.5 8.14193 12.4521 8.26066 12.3562 8.35638C12.2604 8.45211 12.1417 8.49997 12 8.49997H8.49997V12C8.49997 12.1417 8.45203 12.2604 8.35617 12.3562C8.26029 12.4521 8.14149 12.5 7.99977 12.5Z"
+          fill="#00242C"
```

Comment:
Should you use a color from the theme?

Reply:
Renamed to currentColor as used in other icons